lizard_efcis/serializers.py

```diff
@@ -58,7 +67,8 @@ class Meta:
                   'detectiegrens', 'url',
                   'datum', 'tijd',
                   'par_oms',
-                  'validatiestatus')
+                  'validatiestatus',
+                  'eenheid_oms', 'hoed_oms', 'comp_oms')
```

Comment:
Even kijken of je in de views.py ook de .select_related() nog uit moet breiden. Even met de django debug toolbar aan naar de /api pagina voor opnames gaan en dan zie je vanzelf of het nodig is :-)